java-manta-client-unshaded/src/main/java/com/joyent/manta/client/MantaClient.java

```diff
@@ -896,9 +1233,9 @@ public MantaObjectResponse head(final String rawPath) throws IOException {
     }
 
     /**
-     * Get the options associated with a Manta object.
+     * Get the allowed methods associated with a Manta bucket.
```

Comment:
This function as written isn't buckets specific, right?  It still works for any path?

Reply:
It is buckets specific and is valid only for the path specified in javadocs.